api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2325,15 +2325,28 @@ message GetAssistantMessagesResponse {
 message GetAssistantConversationsRequest {}
 
 message ConversationInfo {
+  // id is a unique conversation ID.
   string id = 1;
+  // title is a title of the conversation.
   string title = 2;
+  // createdTime is the time when the conversation was created.
+  google.protobuf.Timestamp created_time = 3 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
 }
 
 message GetAssistantConversationsResponse {
   repeated ConversationInfo conversations = 1;
 }
 
-message CreateAssistantConversationRequest {}
+message CreateAssistantConversationRequest {
+  // createdTime is the time when the conversation was created.
+  google.protobuf.Timestamp created_time = 1 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+}
```

Comment:
Our current conventions for proto files ask for camel casing with a capital start. Can we update the field names in defs/comments to be consistent? this may require updating some existing stuff but feel free to do so, you'd be blocked merging to master without fixing this anyhow.

Reply:
resolved as per the proto3 styleguide. I still find it weird we're mixing styles but I get why.